addons/infinite-scroll/buttonScroll.js

```diff
@@ -14,12 +15,13 @@ async function commentLoader(addon, heightControl, selector) {
 }
 
 export default async function ({ addon, global, console }) {
-  if (
-    window.location.pathname.split("/")[1] === "studios" &&
-    window.location.pathname.split("/")[3] === "comments" &&
-    addon.settings.get("studioScroll")
-  )
-    commentLoader(addon, "#content", "#comments > div:nth-child(2) > ul > div");
+  if (window.location.pathname.split("/")[1].startsWith("studios") && addon.settings.get("studioScroll")) {
```

Comment:
`startsWith` could be `===` just fine, I assume this was to test it properly. Will still work ofc